packages/qwik/src/core/render/jsx/types/jsx-generated.ts

```diff
@@ -582,11 +582,17 @@ type SpecialAttrs = {
     children?: undefined;
   };
   input: {
-    autoComplete?:
-      | HTMLInputAutocompleteAttribute
-      | Omit<HTMLInputAutocompleteAttribute, string>
+    /** 
+     * For type: 'date' | 'datetime-local' | 'email' | 'month' | 'number' | 'password' | 'search' | 'tel' | 'text' | 'time' | 'url' | 'week'
+     * (autocomplete is not application on type: 'button' | 'checkbox' | 'color' | 'file' | 'hidden' | 'image' | 'radio' | 'range' | 'reset' | 'submit')
```

Comment:
"not applicable". I think it's better to just keep the list of non-applicable ones